units/systemd-nspawn@.service.in

```diff
@@ -13,7 +13,7 @@ Before=machines.target
 After=network.target
 
 [Service]
-ExecStart=@bindir@/systemd-nspawn --quiet --keep-unit --boot --link-journal=try-guest --network-veth --settings=override --machine=%i
+ExecStart=@bindir@/systemd-nspawn --quiet --keep-unit --boot --link-journal=try-guest --network-veth -U --settings=override --machine=%i
```

Comment:
@fsateler well, it would have to be `--private-users=try-pick`, `--private-users=try-yes`, `--private-users=try-65536:65536` if you want to be that corect. And I don't like that I must say.

I think saying that if people explicitly configure some feature things fail if the feature is not available is the right thing. I also think it makes sense to provide a shortcut for some features, that behaves a tiny bit more magic, and tries to do the right thing depending on what's available is a good idea, too.